rstudio/rhel9-python-3.9/Dockerfile

```diff
@@ -71,6 +71,8 @@ COPY rstudio/rhel9-python-3.9/rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
 RUN R -e "install.packages('Rcpp')"
+ARG INSTALL_ADD_PKGS
```

Comment:
Could you please add this on the top of the Dockerfile as we do with the rest ARGs?